datascience_eda/datascience_eda.py

```diff
@@ -27,6 +28,10 @@
 from wordcloud import WordCloud, STOPWORDS
 # endregion
 
+# import libraries for eda of categorical features
+import seaborn as sns
```

Comment:
Have you imported the dependency using poetry?

Reply:
No